Tests/DangerTests/BitBucketServerTests.swift

```diff
@@ -9,23 +9,55 @@ final class BitBucketServerTests: XCTestCase {
 
     func testItParsesTheBitBucketPullRequest() {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function body sho...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/BitBucketServerTests.swift;
  Line: 10;
-->

- :warning: Function body should span 40 lines or less excluding comments and whitespace: currently spans 51 lines (`function_body_length`)